serialport.js

```diff
@@ -278,12 +278,12 @@ function SerialPortFactory(_spfOptions) {
   };
 
   SerialPort.prototype.isOpen = function() {
-    return (this.fd ? true : false);
+    return (this.fd !== null ? true : false);
```

Comment:
IMO this would be more clear as:

    return this.fd !== null;

since the conversion to a boolean value is implicit.

Also using `!this.isOpen()` or `!self.isOpen()` instead of replicating the specific value check wherever openness is tested would be more robust.